n3fit/src/n3fit/scripts/evolven3fit_new.py

```diff
@@ -0,0 +1,376 @@
+                        4.86441384506059e-01,
+                        4.94560274153348e-01,
+                        5.02687589545177e-01,
+                        5.10823085439086e-01,
+                        5.18966526903235e-01,
+                        5.27117688756998e-01,
+                        5.35276355048428e-01,
+                        5.43442318565661e-01,
+                        5.51615380379768e-01,
+                        5.59795349416641e-01,
+                        5.67982042055800e-01,
+                        5.76175281754088e-01,
+                        5.84374898692498e-01,
+                        5.92580729444440e-01,
+                        6.00792616663950e-01,
+                        6.09010408792398e-01,
+                        6.17233959782450e-01,
+                        6.25463128838069e-01,
+                        6.33697780169485e-01,
+                        6.41937782762089e-01,
+                        6.50183010158361e-01,
+                        6.58433340251944e-01,
+                        6.66688655093089e-01,
+                        6.74948840704708e-01,
+                        6.83213786908386e-01,
+                        6.91483387159697e-01,
+                        6.99757538392251e-01,
+                        7.08036140869916e-01,
+                        7.16319098046733e-01,
+                        7.24606316434025e-01,
+                        7.32897705474271e-01,
+                        7.41193177421404e-01,
+                        7.49492647227008e-01,
+                        7.57796032432224e-01,
+                        7.66103253064927e-01,
+                        7.74414231541921e-01,
+                        7.82728892575836e-01,
+                        7.91047163086478e-01,
+                        7.99368972116378e-01,
+                        8.07694250750291e-01,
+                        8.16022932038457e-01,
+                        8.24354950923382e-01,
+                        8.32690244169987e-01,
+                        8.41028750298844e-01,
+                        8.49370409522600e-01,
+                        8.57715163684985e-01,
+                        8.66062956202683e-01,
+                        8.74413732009721e-01,
+                        8.82767437504206e-01,
+                        8.91124020497459e-01,
+                        8.99483430165226e-01,
+                        9.07845617001021e-01,
+                        9.16210532771399e-01,
+                        9.24578130473112e-01,
+                        9.32948364292029e-01,
+                        9.41321189563734e-01,
+                        9.49696562735755e-01,
+                        9.58074441331298e-01,
+                        9.66454783914439e-01,
+                        9.74837550056705e-01,
+                        9.83222700304978e-01,
+                        9.91610196150662e-01,
+                        1.00000000000000e00,
+                    ]
```

Comment:
Another infinite value, that should be either generated by a coincise call to something else, or stored as a constant.